main.go

```diff
@@ -24,16 +63,19 @@ func main() {
 	}
 	addr := fmt.Sprint(host, ":", port)
 
-	finish := make(chan struct{})
-
 	header := http.Header{}
-	events, errs := listen.RetryingWatch(url, header, finish)
+	ringEvents, ringErrs := listen.RetryingWatch(ringSubscribeUrl, header, nil)
+	logEvents, logErrs := listen.RetryingWatch(logSubscribeUrl, header, nil)
 
 	go func() {
-		defer close(finish)
+		for err := range ringErrs {
+			log.Printf("Error with ring event stream: %v", err)
+		}
+	}()
 
-		for err := range errs {
-			log.Printf("Error in hookbot event stream: %v", err)
+	go func() {
+		for err := range logErrs {
+			log.Printf("Error with log hookbot event stream: %v", err)
 		}
 	}()
```

Comment:
@scraperdragon 

What you wrote was fine like this (I know you copied it from some other code we have), but you might like to know that you can in principle avoid extra goroutines like so:

```go
go func() {
  for {
    switch {
    case err := <-logErrs:
      log.Printf("Error with log hookbot event stream: %v", err)
    case err := <-ringErrs:
      log.Printf("Error with ring event stream: %v", err)
    }
  }
}
```

You can also do the same with the events, too. This also has the effect of serializing them.